hugo/content/blog/2025-07-11-mumble-1-5-817.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Mumble 1.5.817
+author: Hartmnt
+date: 2025-07-11
+categories:
+  - Release
+---
+
+Today we release the second patch-release in the Mumble 1.5.x series. This release includes fixes for undesired
```

Comment:
IMO we should use the more appropriate *patch release* over the constructed *patch-release*.

Did you make a deliberate decision to combine it into one word?